frame/contracts/src/weights.rs

```diff
@@ -153,1251 +153,1321 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	fn on_initialize() -> Weight {
-		(3_850_000 as Weight)
+		(3_789_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 	}
 	fn on_initialize_per_trie_key(k: u32, ) -> Weight {
-		(52_925_000 as Weight)
-			// Standard Error: 5_000
-			.saturating_add((2_297_000 as Weight).saturating_mul(k as Weight))
+		(0 as Weight)
+			// Standard Error: 3_000
+			.saturating_add((2_332_000 as Weight).saturating_mul(k as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(k as Weight)))
 	}
 	fn on_initialize_per_queue_item(q: u32, ) -> Weight {
-		(434_698_000 as Weight)
-			// Standard Error: 210_000
-			.saturating_add((166_559_000 as Weight).saturating_mul(q as Weight))
+		(0 as Weight)
+			// Standard Error: 17_000
+			.saturating_add((34_838_000 as Weight).saturating_mul(q as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn instrument(c: u32, ) -> Weight {
-		(29_918_000 as Weight)
-			// Standard Error: 185_000
-			.saturating_add((123_774_000 as Weight).saturating_mul(c as Weight))
+		(0 as Weight)
+			// Standard Error: 242_000
+			.saturating_add((2_317_462_000 as Weight).saturating_mul(c as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
```

Comment:
The increased instrument costs are already on master. Not sure what caused this issue. Maybe an updated pwasm-utils?